src/api/workers/enrichment.js

```diff
@@ -51,19 +51,27 @@ const worker = async (id, ctx) => {
 
     return async entry => {
         const enricherTransformer = await transformerFactory(id);
+        const enrichment = await ctx.enrichment.findOneById(id);
+        await ctx.dataset.updateOne(
+            { _id: new ObjectId(entry._id) },
+            { $set: { [enrichment.name]: 'En attente' } },
```

Comment:
What about english translation for En attente?

Reply:
I didn't know how to manage the translation on the API side. Do I have to send the language used from the front end in the request and do the translation accordingly?